packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -188,6 +192,14 @@ class DatePicker extends React.Component {
     this.isDefaultDateAcceptable = this.validateDefaultDate();
   }
 
+  handleBreakpointChange(activeBreakpoint) {
```

Comment:
Can we add some documentation for this function?

Reply:
As of now, we haven't add much, if any, inline documentation in Terra. To me, this is simple enough, the code is describe what the function is doing, like many other places in the library. I actually like that we don't have inline docs everywhere. Inline docs do add value for helpers, scripts or complex function, but to me it makes the code hard to read when its for simple function like this.

I think it's worth opening up a discussion on creating a standard for inline docs and then backfilling the existing Terra components to have them, but if I add it to just this function, it'd be a one-off in this file and I don't think it should be added unless we intend to backfill all fo the functions, which is out of scope of this PR.

Others thoughts?

Reply:
This probably falls into the personal opinion/perspective category, but lack of documentation in other areas of code doesn't mean we shouldn't document any additional code added thereafter.  Adding code to one function doesn't necessitate backfilling every other function in the file.

Inline code documentation is different than documenting the API of a function. This function is rather simple, but it is accepting a parameter and doing something with that parameter.

Private API documentation has always been at the discretion of the developer. Because these functions are private I don't see any harm in either approach. Keeping private documentation at the discretion of the engineer may be the decision that's reached, but I would disagree with discouraging API documentation as a standard for those who choose to do so.

Reply:
> Adding code to one function doesn't necessitate backfilling every other function in the file.

My opinion on this is probably reflect my OCD tendency for the code/structure to be consistent in file (and project) the code is in. 

> but I would disagree with discouraging documentation as a standard for those who choose to do so.

I didn't mean to indicate we should set a standard to not add document. My thought are, if we want to start pushing to adding in-line documentation, it would be helpful to have an agreement on when/how so it's consistent.